### PR TITLE
feat: Added setting to output slow_query

### PR DESCRIPTION
**Pull request by takakuda**
# Description

https://www.notion.so/shinonome-inc/0x-order-watcher-DB-013050aafcad44c7a454fb273713cc73

# Testing Instructions

# Checklist

-   [ ] Update [documentation](https://github.com/0xProject/website/blob/development/mdx/api/index.mdx) as needed. **Website Documentation PR:**
-   [ ] Prefix PR title with `[WIP]` if necessary.
-   [ ] Add tests to cover changes as needed.
-   [ ] Test changes on the staging environment with [Matcha API Staging](https://api-staging.matcha.xyz).

    -   [ ] SRA/Limit orders
    -   [ ] Swap endpoints
    -   [ ] Meta transaction endpoints
    -   [ ] Depth charts

    For more information see `0x API Matcha smoke test runbook` in Quip.

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -19,4 +19,5 @@ services:
         volumes:
             - ./postgres:/var/lib/postgresql/data
         ports:
-            - '5432:5432'
\ No newline at end of file
+            - '5432:5432'
+        command: postgres -c log_min_duration_statement=3000 -c logging_collector=on -c log_destination=stderr -c log_statement=all
```
Comment (takakuda) [inline docker-compose.yml:23]: - log_min_duration_statement
3秒以上かかるクエリだったら`/var/lib/postgresql/data/pg_log`に対象のslow queryが出力される

- logging_collector
標準出力をログファイルにリダイレクトするか否か

- log_destination
logの出力先

- log_statement
どのSQL文をlogに出力するか

**Comment by takakuda** [comment]
```
# cat postgresql-2022-12-21_140244.log
LOG:  database system was shut down at 2022-12-14 03:54:50 UTC
LOG:  MultiXact member wraparound protections are now enabled
LOG:  autovacuum launcher started
LOG:  database system is ready to accept connections
WARNING:  could not open statistics file "pg_stat_tmp/global.stat": Operation not permitted
FATAL:  role "root" does not exist
FATAL:  password authentication failed for user "api"
DETAIL:  Password does not match for user "api".
	Connection matched pg_hba.conf line 95: "host all all all md5"
ERROR:  syntax error at or near "api" at character 1
STATEMENT:  api
	show all;
LOG:  statement: show all;
LOG:  statement: select pg_sleep(4);
LOG:  duration: 4030.987 ms
LOG:  statement: select pg_sleep(10);
LOG:  duration: 10014.980 ms
WARNING:  could not open statistics file "pg_stat_tmp/global.stat": Operation not permitted
FATAL:  role "root" does not exist
LOG:  statement: select pg_sleep(5);
LOG:  duration: 5007.715 ms
LOG:  statement: select pg_sleep(2);
```

3秒以上かかったqueryがlogに出力されています。

**Comment by massun-onibakuchi** [comment]
lg

**Comment by ghost** [comment]
lg

**Pull request merged**